src/mbus-master.cc

```diff
@@ -254,12 +254,15 @@ class RecieveWorker : public Nan::AsyncWorker {
 
         memset((void *)&reply, 0, sizeof(mbus_frame));
 
-        if (init_slaves(handle) == 0)
+        if (ping_first)
```

Comment:
I think this could be simplified as follows to get rid of one level of nesting:

```suggestion
        if (ping_first && init_slaves(handle) == 0)
```

Reply:
Will also adjust after merge